File: android-twa/app/src/main/java/app/polycal/widgets/NlpEventWidget.java
package app.polycal.widgets;

import app.polycal.R;

/** Home-screen NLP compose bar (PC-479). */
public class NlpEventWidget extends ComposeAppWidget {
    @Override
    String mode() {
        return WidgetUrls.MODE_NLP;
    }

    @Override
    int layoutRes() {
        return R.layout.widget_nlp_event;
    }

    @Override
    int headerId() {
        return R.id.widget_header;
    }

    @Override
    int inputId() {
        return R.id.widget_input;
    }

    @Override
    int sendId() {
        return R.id.widget_send;
    }

    @Override
    int rootId() {
        return R.id.widget_root;
    }
}

File: android-widgets/app/src/main/java/app/polycal/widgets/NlpEventWidget.kt
package app.polycal.widgets

/** Homescreen widget: one-line description → New Event (NLP Input) composer. */
class NlpEventWidget : ComposeAppWidget() {
    override val mode = WidgetUrls.MODE_NLP
    override val layoutRes = R.layout.widget_nlp_event
    override val headerId = R.id.widget_header
    override val inputId = R.id.widget_input
    override val sendId = R.id.widget_send
    override val rootId = R.id.widget_root
}
